projects/architecture/interfaces/primitives/addresses.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+message UniversalAddress {
```

Comment:
This will change, according to the last discussions with architecture. I'll address its affects on the SDK and UniversalAddress in a future PR.